src/co/elastic/NotificationManager.groovy

```diff
@@ -414,3 +417,35 @@ def queryFilter(jobName) {
 }
   """
 }
+
+
+/**
+ * This method searches for the IssueCommentTrigger in the project itself
+ * and if so, then look for the GitHub comment trigger which it's supported.
+ */
+def issueCommentTrigger() {
+  def issueCommentTrigger = findIssueCommentTrigger()
+
+  if (issueCommentTrigger == null) {
+    log(level: 'WARN', text: "No comment bad happened when creating")
```

Comment:
`bad happened`? Not sure if `has / had` happened :)

Reply:
```suggestion
    log(level: 'WARN', text: "No comment has happened when creating")
```